hazelcast-client/src/test/java/com/hazelcast/client/test/HazelcastFactoryTest.java

```diff
@@ -33,17 +35,27 @@
 @Category({QuickTest.class, ParallelTest.class})
 public class HazelcastFactoryTest extends HazelcastTestSupport {
 
+    private TestHazelcastFactory instanceFactory;
```

Comment:
I pulled out the `TestHazelcastFactory` to shut it down even if an exception is thrown in the tests.